akka-remote/src/main/scala/akka/remote/artery/ArterySubsystem.scala

```diff
@@ -57,6 +68,11 @@ private[remote] class ArterySubsystem(_system: ExtendedActorSystem, _provider: R
     else Future.successful(Done)
   }
 
+  def sendReply(to: Address, message: AnyRef) = {
```

Comment:
sure, I'm even using this `dummyRecipient` hack now, but I have sprinkled fixme about that. somehow we need to be able to send back a message to the originAddress